cmd/prstatus/prstatus_test.go

```diff
@@ -13,7 +13,7 @@
  *
  */
 
-package pr_status
+package prstatus
```

Comment:
This is because golang recommends package names don't have underscores and gofumpt was nagging me.